src/chain.h

```diff
@@ -370,6 +371,8 @@ class CDiskBlockIndex : public CBlockIndex
         hashPrev = (pprev ? pprev->GetBlockHash() : uint256());
     }
 
+    friend bool operator<(const CDiskBlockIndex &a, const CDiskBlockIndex &b) { return a.nHeight < b.nHeight; }
+    friend bool operator>(const CDiskBlockIndex &a, const CDiskBlockIndex &b) { return a.nHeight > b.nHeight; }
```

Comment:
How are these used?  Is their use consistent with the possibility that nHeight might be the same (orphan block)

Reply:
we only use this to run std::sort() so we can sort the indexes with some accuracy when we prepare to upgrade/downgrade the blockdb